sapp/ui/filter_predicates.py

```diff
@@ -97,46 +108,42 @@ def apply(self, issues: List[IssueQueryResult]) -> List[IssueQueryResult]:
         ]
 
 
+class Matches(IssuePredicate):
+    def __init__(self, regex: str, parameter_name: str) -> None:
+        self._regex: Pattern[str] = js_regex.compile(regex)
+        self._parameter_name = parameter_name
+
+    def attribute_set(self, issue: IssueQueryResult) -> Set[str]:
+        attribute = issue._asdict()[self._parameter_name]
+        if type(attribute) is str:
```

Comment:
Here and other places: I would prefer `isinstance(attribute, str)` over `type(attribute) is str`.

Reply:
Sure